js/modules/k6/http/response.go

```diff
@@ -68,17 +67,12 @@ func (res *Response) JSON(selector ...string) goja.Value {
 
 // HTML returns the body as an html.Selection
 func (res *Response) HTML(selector ...string) html.Selection {
-	var body string
-	switch b := res.Body.(type) {
-	case []byte:
-		body = string(b)
-	case string:
-		body = b
-	default:
-		common.Throw(common.GetRuntime(res.GetCtx()), errors.New("invalid response type"))
+	body, err := common.ToBytes(res.Body)
+	if err != nil {
+		common.Throw(common.GetRuntime(res.GetCtx()), err)
 	}
 
-	sel, err := html.HTML{}.ParseHTML(res.GetCtx(), body)
+	sel, err := html.HTML{}.ParseHTML(res.GetCtx(), string(body))
```

Comment:
This seems to me that will go string>[]byte-> string in the more common case IMO.

Maybe `ToString` ?

Reply:
Nice catch ;) I didn't think it was important to optimize, but sure, we could add a `ToString` function.

Reply:
Resolved by e84daccb.